sdk/containerservice/Azure.ResourceManager.ContainerService/samples/Generated/Samples/Sample_ContainerServiceManagedClusterResource.cs

```diff
@@ -251,7 +251,7 @@ public async Task ResetServicePrincipalProfile_ResetServicePrincipalProfile()
         public async Task ResetAadProfile_ResetAADProfile()
         {
             // Generated from example definition: specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-10-01/examples/ManagedClustersResetAADProfile.json
-            // this example is just showing the usage of "ManagedClusters_ResetAadProfile" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "ManagedClusters_ResetAADProfile" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `AAD`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-09-01/managedClusters.json#L1501